packages/react-scheduler/src/__tests__/ReactScheduler-test.js

```diff
@@ -361,5 +393,354 @@ describe('ReactScheduler', () => {
     });
   });
 
+  describe('when callbacks throw errors', () => {
+    describe('when some callbacks throw', () => {
+      /**
+       * +                                                             +
+       * |  rAF                        postMessage                     |
+       * |                                                             |
+       * |      +---------------------+                                |
+       * |      | paint/layout        |  cbA() cbB() cbC() cbD() cbE() |
+       * |      +---------------------+         ^           ^          |
+       * |                                      |           |          |
+       * +                                      |           |          +
+       *                                        +           +
+       *                                        throw errors
+       *
+       *
+       */
```

Comment:
I'm a little confused about the graph, if you call poseMessage, the onmessage handler is fired **after** microtask and **before** rAF, but this graph shows it will be fired after rAF, why?